e2e/cephfs.go

```diff
@@ -164,6 +164,9 @@ var _ = Describe("cephfs", func() {
 			logsCSIPods("app=csi-cephfsplugin-provisioner", c)
 			// log node plugin
 			logsCSIPods("app=csi-cephfsplugin", c)
+
+			// log all details from the namespace where Ceph-CSI is deployed
+			framework.DumpAllNamespaceInfo(c, cephCSINamespace)
```

Comment:
this will log all pods in the namespace and the node information. as we are already logging the CSI pods is this useful?

Reply:
We are currently not logging enough to identify problems with the Vault deployment. If the `logCSIPods()` function logs the same details, just leaving `DumpAllNamespaceInfo()` might be sufficient?

Reply:
Yeah, that would be good, can you remove the current logging and use DumpAllNamespaceInfo and also make sure you make some code changes in code so E2E fails. we can verify that logging works.

Reply:
Updated with replacement, will add a commit that causes a failure if things pass this run.

Reply:
Right, so these CI jobs all succeeded. Now added a forced failure when deleting pods, that should show the functioning of the change in logging.

Reply:
`DumpAllNamespaceInfo()` does not include the logs of pods. It seems `logCSIPods()` is still helpful, so I will re-add that.